flask/app/routes.py

```diff
@@ -179,33 +179,49 @@ def mixin(entity: db.Model, json_mixin: Dict[str, Any], columns: List[str]) -> U
             setattr(entity, field, value)
 
 
-@app.route('/api/participants/<int:participant_id>', methods = ['PATCH'])
+@app.route('/api/<model_name>/<int:id>', methods = ['PATCH'])
 @login_required
-def update_participants(participant_id: int):
+def update_db(model_name:str, id:int):
     if not request.json:
         return 'Request body must be JSON', 415
 
-    participant = models.Participant.query.get(participant_id)
-    if not participant:
+    editable_columns = []
```

Comment:
Don't believe this is necessary either because the `else` branch returns out of the function and we're not doing the same for `table` either